app/helpers/application_helper.rb

```diff
@@ -163,22 +163,26 @@ def boolean_radio_buttons_collection(text_vals = { true: 'Yes', false: 'No' })
   def expire_date_label_and_value(entity)
```

Comment:
This method doesn't belong in this class.   The responsibility of this class is for view-related functionality that is or can be applied to all views in the application.  This method applies only views connected to Users and Companies (and really, only to those things that have _expiration dates_).

This should be refactored out.  Commonality should be abstracted into a common class or module.
The UsersHelper and CompaniesHelper classes can then incorporate them (via `include` or whatever) as needed.